string-manipulation-1/src/get-first-char.js

```diff
@@ -0,0 +1,13 @@
+/* exported getFirstChar */
+
+function getFirstChar(string) {
+  for (var i = 0; i < string.length; i++) {
+    if (i === 0) {
+      return string[i];
+    }
+  }
+}
```

Comment:
You don't need a loop here. You can just return `string[0]`

Reply:
🫡